rllib/algorithms/impala/tf/impala_tf_policy_rlm.py

```diff
@@ -121,12 +121,12 @@ def loss(
 
         # The policy gradients loss.
         pi_loss = -tf.reduce_sum(target_actions_logp_time_major * pg_advantages)
-        mean_pi_loss = -tf.reduce_mean(target_actions_logp_time_major * pg_advantages)
+        mean_pi_loss = pi_loss
 
         # The baseline loss.
         delta = values_time_major - vtrace_adjusted_target_values
         vf_loss = 0.5 * tf.reduce_sum(tf.math.pow(delta, 2.0))
-        mean_vf_loss = 0.5 * tf.reduce_mean(tf.math.pow(delta, 2.0))
+        mean_vf_loss = vf_loss
```

Comment:
@avnishn Why are we doing this? I know we'll remove this eventually. I'm still curious if this is an oversight or if the means are deliberately not calculated.

Reply:
This is deliberate as far as I can tell, to minimize the old and new differences.